python/pyhail/dataset.py

```diff
@@ -382,7 +382,56 @@ def annotate_variants_bed(self, input, root, all=False):
     def annotate_variants_expr(self, condition):
         """Annotate variants with expression.
 
-        :param str condition: Annotation expression.
+        **Examples**
+
+        Compute GQ statistics about heterozygotes per variant
+
+        >>> (vds.annotate_variants_expr('va.gqHetStats = '
```

Comment:
I was assuming `hc` was in scope, but that's it.  Should we also assume `vds = hc.read('data/example.vds')` is in scope, too?